microsoft-edge/webview2/get-started/winforms.md

```diff
@@ -28,19 +28,25 @@ Install the following list of prerequisites before proceeding.
 
 Start with a basic desktop project that contains a single main window.  
 
-1.  In Visual Studio, choose **Windows Forms .NET Framework App** > **Next**.
+1. Open Visual Studio. In the opening panel, choose **Create a new project**.
+
+    :::image type="complex" source="./media/winforms-opening-panel.png" alt-text="Visual Studio opening panel" lightbox="./media/winforms-opening-panel.png":::
+       Visual Studio opening panel  
+    :::image-end:::
+
+1. Locate and to choose **C# Windows Forms .NET Framework App** then choose **Next**.
     
     :::image type="complex" source="./media/winforms-new-project.png" alt-text="New project" lightbox="./media/winforms-new-project.png":::
        New project  
     :::image-end:::
     
-1.  Enter values for **Project name** and **Location**.  Choose **.NET Framework 4.6.2** or later.  
+1.  Enter values for **Project name** and **Location**.  Choose **.NET Framework 4.7.2** or later.  
     
     :::image type="complex" source="./media/winforms-start-proj.png" alt-text="Start project" lightbox="./media/winforms-start-proj.png":::
        Start project  
     :::image-end:::
     
-1.  To create your project, choose **Create**.
+1.  Choose **Create**.
```

Comment:
Global in this .md file: Change 'choose' to 'select'. 
Source: public (or internal) version of MWSG, article "Describing interactions with UI", row: "Select" (& row "Choose"). 
 https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui
Mgmt approved, in a previous conversation.